lib/connect.ts

```diff
@@ -170,7 +170,7 @@ var KiteConnect = function(params) {
     requestInstance.interceptors.response.use(function(response) {
         if (self.debug) console.log(response);
 
-        var contentType = response.headers["content-type"];
+        const contentType = response.headers["content-type"];
         if (contentType === "application/json" && typeof response.data === "object") {
```

Comment:
@bhavya3024 Just one Suggestion, can we make one constant file and maintain this string constant
ex. `if (contentType === "application/json" && typeof response.data === CONSTANT.APPLICATION_JSON)`